pkg/quay/quay_test.go

```diff
@@ -20,199 +20,461 @@ import (
 	"fmt"
 	"net/http"
 	"reflect"
-	"strings"
 	"testing"
 
+	"gotest.tools/v3/assert"
+
 	"github.com/h2non/gock"
 )
 
 const (
 	org       = "test_org"
 	repo      = "test_repo"
 	robotName = "robot_name"
+
+	testRepoName        = "test-repo-using-api"
```

Comment:
Is there a need for `testRepoName` instead of using `repo`?

Reply:
Good catch. This was refactored from corresponding test. Updated to use `repo`.